.azure_pipelines/ci-pipeline-makefile.yml

```diff
@@ -58,6 +58,7 @@ jobs:
       # run all tests
       - script: |
           make -j tests ALLTESTS=1
+          make -C tests clean
```

Comment:
The 'solutions' are the bigger spender of disk spaces. Also the clean up should be only enabled for pipeline runs. We do want to keep the intermediate files for local runs in case we need to investigate failures.

Reply:
`make clean` will only run for pipeline. It shouldn't affect local runs.

In this case, cleaning tests makes enough room for solution tests